transformer_class.py

```diff
@@ -0,0 +1,200 @@
+"""Class instance for Transformer
+"""
+
+import os
+import argparse
+import datetime
+import logging
+import piexif
+
+import configuration
+
+# EXIF tags to look for
```

Comment:
```suggestion
# EXIF tags to look for, see https://www.exiv2.org/tags.html
```

36867 is linked to a tag called "Exif.Image.DateTimeOriginal" ... I don't want to get too picky on this but is ORIGIN a typo (did you mean EXIF_ORIGINAL_TIMESTAMP)  but are these consistent with the EXIF standard, i.e. is "The date and time when the original image data was generated. For a digital still camera the date and time the picture was taken are recorded." an accurate description of the EXIF_ORIGIN_TIMESTAMP variable?

Reply:
Please be picky. It looks like it's a typo - making change